README.md

```diff
@@ -22,17 +25,37 @@ You can use the following options:
 - Any of the options supported by `Ecto.Repo` itself, which you can see
       in the `Ecto.Repo` documentation.
 
-- Any of the option supported by `Xandra.Cluster.start_link/1`.
+- Any of the options supported by `Xandra.Cluster.start_link/1`.
 
 To configure your Ecto repository to use this adapter, you can use the
   `:adapter` option. For example, when defining the repo:
 
 ```elixir
 defmodule MyApp.Repo do
-  use Ecto.Repo, otp_app: :my_app, adapter: Exandra
+  use Ecto.Repo,
+    otp_app: :my_app,
+    adapter: Exandra
```

Comment:
Can you revert this change, as it's unnecessary?

Reply:
done